### ISSUE TITLE
Opening on-screen keyboard hides text entry field

### ISSUE THREAD

**Issue by yuttadhammo** [Sirimangalo/meditation-plus-server#7]
Only on mobile Firefox, not sure if this is fixable, and not a big issue because you can scroll down, but clocking on the time entry fields hides them behind the keyboard rather than moving the entry fields up. Chrome for Android has no problem.

**Comment by binarious** [comment]
I don't know what to do about that. Have you noticed this behaviour on another site, too?

**Comment by yuttadhammo** [comment]
No, that's why i mention it... Never had this problem elsewhere. It's okay, just an inconvenience

**Comment by binarious** [comment]
Ok, I would suggest adding it to the Homescreen via Chrome since it is a Progressive Webapp which is best supported by Chrome.
